### ISSUE TITLE
Murphpad Schematic.

### ISSUE THREAD

**Issue by tbandtg** [MechWild/public-design-files#2]
Is there a schematic for the board for the murphad, that would be a great asset to have. For trouble shooting as well as just looking at other micro options.

**Comment by bentorkington** [comment]
Yeah, I'd like this too. My pad seems to work, but no LEDs. I *guess* LED1 is the first in the chain, but it'd be good to know for sure. Please?
